Tabloid/client/src/modules/tagManager.js

```diff
@@ -13,12 +13,29 @@ export const getAllTags = () => {
             if (res.ok) {
                 return res.json();
             } else {
-                throw new Error("Unknown error getting tags");
+                throw new Error("Unknown Error Getting Tags");
             }
         });
     });
 };
 
+export const getTagById = (id) => {
+    return getToken().then((token) => {
+      return fetch(`${_apiUrl}/${id}`, {
+        method: "GET",
+        headers: {
+          Authorization: `Bearer ${token}`,
+        },
+      }).then((res) => {
+        if (res.ok) {
+          return res.json();
+        } else {
+          throw new Error("Unknown Error Getting Tags");
```

Comment:
Is this an error getting tags or a single tag?

Reply:
Fixed